src/react-raf-shim.js

```diff
@@ -0,0 +1,5 @@
+const raf = global.requestAnimationFrame = (cb) => {
```

Comment:
if this is only for tests, maybe it makes sense to create a _shim_ folder in the test directory?  Also, maybe a comment to a link or github issue providing an explanation for why this is needed?